specification/compute/resource-manager/Microsoft.Compute/stable/2021-11-01/compute.json

```diff
@@ -8113,6 +8113,60 @@
           "final-state-via": "azure-async-operation"
         }
       }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Compute/hostGroups/{hostGroupName}/hosts/{hostName}/reboot": {
+      "post": {
+        "tags": [
+          "DedicatedHost"
+        ],
+        "operationId": "DedicatedHosts_Reboot",
+        "description": "Reboot the dedicated host. The operation will complete successfully once the dedicated host has booted and is running. For VM health after the reboot, please check the Resource Health Center in the Azure Portal.",
```

Comment:
We need to add a doc link that points to how a user can look a the the VM resource health in RHC.

Reply:
Sure, I'll refer the user to this URL.